components/support/migration/src/main/java/mozilla/components/support/migration/TelemetryHelpers.kt

```diff
@@ -16,6 +16,7 @@ internal fun Migration.telemetryIdentifier(): String {
         Migration.Settings -> "settings"
         Migration.Addons -> "addons"
         Migration.TelemetryIdentifiers -> "telemetry_identifiers"
+        Migration.SearchEngine -> "search_engine"
```

Comment:
in metrics.yaml `migration_versions` you called it `search` - these labels need to be the same (also, see comment on top of this function).

Reply:
Argh, wrote that before any other telemetry code. :)